crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -23,6 +23,13 @@ def status
         ::Crowbar::UpgradeStatus.new.progress
       end
 
+      def node_status
+        {
+          upgraded: ::Node.all.reject(&:admin?).select(&:ready_after_upgrade?).map(&:name),
+          not_upgraded: ::Node.find("state:crowbar_upgrade").map(&:name)
```

Comment:
i guess only when it's executed after the `prepare` step. @jsuchome is this an issue?